Tasks/VsTest/distributedtest.ts

```diff
@@ -68,25 +50,27 @@ export class DistributedTest {
         }
     }
 
-    private async startDtaExecutionHost(agentId: any) {
-        this.testSourcesFile = this.createTestSourcesFile();
+    private async startDtaExecutionHost() {
+        const dtaExecutionHostTool = tl.tool(path.join(__dirname, 'Modules/DTAExecutionHost.exe'));
         const envVars: { [key: string]: string; } = process.env;
```

Comment:
nitpick : can we move this closer to the usage ?

Reply:
Done